colossalai/inference/modeling/models/llama.py

```diff
@@ -39,6 +39,16 @@ def apply_rotary_pos_emb(q, k, cos, sin, position_ids):
     return q_embed, k_embed
 
 
+def get_cos_sin(lengths, cos_cache, sin_cache):
```

Comment:
How can we use this cache in llama model？

Reply:
I will add docsrting

Reply:
done